cpp/cpp_odbc/Library/cpp_odbc/statement.h

```diff
@@ -155,6 +155,14 @@ class statement {
     bool more_results() const;
 
     virtual ~statement();
+
+    /**
+     * @brief Close all odbc ressources associated with this instance
+     * 
+     * This should only be called directly prior to the destructor; it is not
+     * valid to call any other method (except the destructor) after calling finalize.
+     */
+    virtual void finalize();
```

Comment:
What's the reason to make this virtual? It doesn't seem to be overloaded later on.